tests/e2e_logging.py

```diff
@@ -1347,33 +1348,33 @@ def additional_interfaces(local_node_id):
     )
 
     # Is there a better way to do this?
-    if os.path.exists(
-        os.path.join(cr.args.library_dir, "libjs_v8.virtual.so")
-    ) or os.path.exists(os.path.join(cr.args.library_dir, "libjs_v8.enclave.so")):
-        cr.add(
-            "js_v8",
-            run,
-            package="libjs_v8",
-            nodes=infra.e2e_args.max_nodes(cr.args, f=0),
-            initial_user_count=4,
-            initial_member_count=2,
-        )
-
-    cr.add(
-        "cpp",
-        run,
-        package="samples/apps/logging/liblogging",
-        js_app_bundle=None,
-        nodes=infra.e2e_args.max_nodes(cr.args, f=0),
-        initial_user_count=4,
-        initial_member_count=2,
-    )
-
-    cr.add(
-        "common",
-        e2e_common_endpoints.run,
-        package="samples/apps/logging/liblogging",
-        nodes=infra.e2e_args.max_nodes(cr.args, f=0),
-    )
+    # if os.path.exists(
+    #     os.path.join(cr.args.library_dir, "libjs_v8.virtual.so")
+    # ) or os.path.exists(os.path.join(cr.args.library_dir, "libjs_v8.enclave.so")):
+    #     cr.add(
+    #         "js_v8",
+    #         run,
+    #         package="libjs_v8",
+    #         nodes=infra.e2e_args.max_nodes(cr.args, f=0),
+    #         initial_user_count=4,
+    #         initial_member_count=2,
+    #     )
+
+    # cr.add(
+    #     "cpp",
+    #     run,
+    #     package="samples/apps/logging/liblogging",
+    #     js_app_bundle=None,
+    #     nodes=infra.e2e_args.max_nodes(cr.args, f=0),
+    #     initial_user_count=4,
+    #     initial_member_count=2,
+    # )
+
+    # cr.add(
+    #     "common",
+    #     e2e_common_endpoints.run,
+    #     package="samples/apps/logging/liblogging",
+    #     nodes=infra.e2e_args.max_nodes(cr.args, f=0),
+    # )
```

Comment:
To be restored?

Reply:
How did that make it through!?! Sorry about that :-)